models/FeaturedItem.yaml

```diff
@@ -0,0 +1,121 @@
+type: object
+name: FeaturedItem
+label: Featured Item section
+labelField: title
+extends:
+  - Section
+groups:
+  - sectionComponent
+fieldGroups:
+  - name: styles
+    label: Styles
+  - name: settings
+    label: Settings
+fields:
+  - type: string
+    name: title
+    label: Title
+    default: Item
+  - type: string
+    name: subtitle
+    label: Subtitle
+    default: Featured blog posts section example
+  - type: list
+    name: items
+    label: Items
+    items:
+      type: model
+      models:
+        - Item
+    default:
+      - type: Item
+        title: 'Special Item'
+        subtitle: 'Subtitle'
+        content: 'This is the content of the item'
+        author: 'John Doe'
+        rating: 5
+  - type: list
+    name: actions
+    label: Actions
+    items:
+      type: model
+      models:
+        - Button
+        - Link
+    default:
+      - type: Button
+        label: Get Started
+        url: '/'
+        style: primary
+        elementId: hero-main-button
+      - type: Button
+        label: Learn More
+        url: '/'
+        style: secondary
+  - type: style
+    name: styles
+    styles:
+      self:
+        height: ['auto', 'screen']
+        width: ['narrow', 'wide', 'full']
+        margin: ['tw0:36']
+        padding: ['tw4:36']
+        alignItems: ['flex-start', 'flex-end', 'center']
+        justifyContent: ['flex-start', 'flex-end', 'center']
+        flexDirection: ['row', 'row-reverse', 'col', 'col-reverse']
+        borderRadius: '*'
+        borderWidth: ['0:8']
+        borderStyle: '*'
+        borderColor:
+          - value: 'border-primary'
+            label: 'Primary color'
+            styleObjectColor: 'primary'
+          - value: 'border-secondary'
+            label: 'Secondary color'
+            styleObjectColor: 'secondary'
+          - value: 'border-neutral'
+            label: 'Neutral color'
+            styleObjectColor: 'neutral'
+          - value: 'border-complementary'
+            label: 'Complementary color'
+            styleObjectColor: 'complementary'
+          - value: 'border-complementary-alt'
+            label: 'Complementary alt color'
+            styleObjectColor: 'complementaryAlt'
```

Comment:
`styleObjectColor` was removed use this link as example how to update the values -> https://github.com/stackbit/stackbit-components/blob/534876ade3b91f757435222271e67efa1213d066/models/HeroSection.yaml#L82